waspc/src/Wasp/Generator/WebAppGenerator/AuthG.hs

```diff
@@ -29,7 +26,7 @@ genAuth spec =
     Nothing -> return []
     Just auth ->
       sequence
-        [ copyTmplFile [relfile|auth/logout.js|],
+        [ copyTmplFile [relfile|auth/logout.ts|],
           copyTmplFile [relfile|auth/helpers/user.ts|],
           copyTmplFile [relfile|auth/types.ts|],
```

Comment:
Rename `copyTmplFile` to `genFileCopy` to keep naming consistent with other generator files (for example https://github.com/wasp-lang/wasp/blob/main/waspc/src/Wasp/Generator/WebAppGenerator.hs#L76).

I would put a suggestion here like the places in AuthFormsG.hs but GH doesn't let me since some of the lines are unchanged.